### DISCUSSION TITLE
repeating keys on hold

### DISCUSSION THREAD

**Discussion by mankoff** [manna-harbour/miryoku#156] [Q&A] [answered]
I'm using Dvorak layout on a host in Dvorak. Therefore my Miryoku map uses `DV_N`.

When I press and hold CTRL+p for previous line, using either CTRL key, the p repeats and the cursor moves more than once. 

When I press CTRL+n for next line, it only moves 1x. I need to tap repeatedly to get it to move more than one line. 

I'm new to QMK. How should I start debugging this? Or is there a command line compile flag to adjust this?

**Comment by mankoff** [comment]
I see now that this is a side effect of home row mods.

**Comment by manna-harbour** [comment, marked as answer]
There are many tap-hold keys on Base, so in general keys on Base don't auto repeat. For alphas for use in navigation, or for thumbs, use Nav instead which will auto repeat. Double-tap then hold for autorepeat can be enabled manually but will prevent rapid tap followed by tap-hold holds. Also see https://github.com/manna-harbour/miryoku/discussions/73.
